static/css/sass/cdr-ui.scss

```diff
@@ -457,6 +457,19 @@ $border-style: 1px solid $light-gray;
 $box-shadow-color: rgba(211,211,211,0.5);
 $box-shadow: inset 3px 3px 10px -1px $box-shadow-color, 0px 16px 22px -15px $box-shadow-color;
 
+/* Items marked for deletion */
+.marked-delete {
```

Comment:
there are still existing styles for deleted items, they are over in here:
https://github.com/UNC-Libraries/Carolina-Digital-Repository/blob/46e5d59c48bde7288c6c80f348d6480ce86cada5/static/css/cdrui_styles.css#L200-L218
https://github.com/UNC-Libraries/Carolina-Digital-Repository/blob/46e5d59c48bde7288c6c80f348d6480ce86cada5/static/css/cdrui_styles.css#L971-L986

the class name is `deleted`. I don't know if these are still helpful or not, but there seems to be a bit more happening there than in this rule, so its worth comparing. It covers record and result pages. We should probably consider if there are more styles there that we want to retain, and cleanup the old rules once we have everything we want in here. 

Also, `marked-delete` reads a little weird to me, maybe because its an adjective plus verb. Maybe `deleted` or `marked-deleted`?

One other thing, in bxc3 we get a trash icon over the thumbnail when something is deleted. I haven't deployed this branch yet, but i don't see anything in the PR that would add that in.

For an example of a deleted resource, see:
https://dcr-qa.lib.unc.edu/search?anywhere=doomed_from_the_start
you might have to spoof yourself some credentials, I can't remember your permission setup.

Reply:
No, there, currently, isn't any trash icon added.

Reply:
I saw the exisiting classes, but I think they weren't being applied for some reason in the Vue components if a link had the "deleted" class. I'll poke around

Reply:
I was thinking at some point we might want to find/run a tool to find "dead" css.